examples/usecases/multi-gpu-data-parallel-training.ipynb

```diff
@@ -0,0 +1,459 @@
+{
```

Comment:
<p>typo in it's --&gt; "its batch" is the correct one.</p><p> </p><p>"All model parameters fits on a single GPU. do you mean " in this example we assume that model parameters fits on a single GPU." ?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/855/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Correct spelling</p><p>and updating describption</p>